### ISSUE TITLE
UserWarning: masked_select received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead

### ISSUE THREAD

**Issue by kasimte** [RunzheYang/MORL#3]
I repeatedly get this deprecation warning in the logs during training. 

```
../aten/src/ATen/native/LegacyDefinitions.cpp:67: UserWarning: masked_select received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
```

Here is my `pip list`:

```
(morl) Kasim:MORL Kasim$ pip list
Package          Version            
---------------- -------------------
certifi          2019.9.11          
chardet          3.0.4              
idna             2.8                
jsonpatch        1.24               
jsonpointer      2.0                
numpy            1.17.4             
Pillow           6.2.1              
pip              19.3.1             
pyzmq            18.1.1             
requests         2.22.0             
scipy            1.3.3              
setuptools       42.0.1.post20191125
six              1.13.0             
torch            1.3.1              
torchfile        0.1.0              
tornado          6.0.3              
urllib3          1.25.7             
visdom           0.1.8.9            
websocket-client 0.56.0             
wheel            0.33.6             
(morl) Kasim:MORL Kasim$ 
```

Should I be using a different version of `torch`?

**Comment by RunzheYang** [comment]
PyTorch version for `synthetic` was torch 0.4.0, and the visdom version is 0.1.6.3. Please the update readme

**Comment by kasimte** [comment]
Thanks, @RunzheYang! I installed torch `0.4.1` and the console log is much cleaner now.
